editor/src/scene_viewer/gizmo.rs

```diff
@@ -206,6 +210,13 @@ impl SceneGizmo {
             pos_z,
             neg_z,
             center,
+            is_left_mouse_pressed: false,
+            last_mouse_position: None,
+            camera_rotation: CameraRotation {
```

Comment:
This is not needed, all you need to do is to pass a mutable reference to the current camera controller instance and change its `pitch` and `yaw` fields accordingly to the mouse movement - horizontal movement should change the yaw, and the vertical movement should change the pitch.